.github/workflows/issue-to-pr.yaml

```diff
@@ -0,0 +1,189 @@
+          echo '{}' > $jsonfile
+
+          echo "Bumping $namespace"
+          cd src
+          # This might "bump" a few extra providers, but it will only commit the specific provider below
+          go run ./cmd/bump-versions -provider-namespace $namespace -module-namespace this-really-should-not-exist-please-ignore-me
+          cd -
+
+
+          # Validate json file
+          set +e
+          grep -c '"version":' $jsonfile
+          if [[ "$?" != 0 ]]; then
+            gh issue comment $NUMBER -b "Failed validation: No versions detected in $jsonfile"
+            exit 1
+          fi
+          set -e
+
+          
+          git config --global user.email "no-reply@opentofu.org"
+          git config --global user.name "OpenTOFU Automation"
+
+          branch="provider-submission_${namespace}_${name}"
+          set +e
+          git checkout -b $branch
+          if [[ "$?" != 0 ]]; then
+            gh issue comment $NUMBER -b "Failed validation: A branch already exists for this provider '$branch'"
+            exit 1
+          fi
+          set -e
+
+          git add $jsonfile
+          git commit -s -m "Create provider $namespace/$name"
+          git push -u origin $branch
+
+          pr=$(gh pr create --title "$TITLE" --body "Created $jsonfile for provider $namespace/$name.  See issue #$NUMBER for details.") #--assignee opentofu/core-engineers)
+
+          gh issue comment $NUMBER -b "Your submission has been accepted and has moved on to the pull request phase ($pr)"
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GH_REPO: ${{ github.repository }}
+          NUMBER: ${{ github.event.issue.number }}
+          URL: ${{ github.event.issue.url }}
+          TITLE: ${{ github.event.issue.title }}
+          BODY: ${{ github.event.issue.body }}
+
+  module_issue_to_pr:
+    if: contains(github.event.issue.labels.*.name, 'module') && contains(github.event.issue.labels.*.name, 'submission') 
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      contents: write
+      pull-requests: write
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version-file: './src/go.mod'
+      - name: Module Issue to Pull Request
+        run: |
+          # Pull module out of issue body
+          module=$(echo "$BODY" | grep "### Repository" -A2 | tail -n1 | sed -e 's/[\r\n]//g')
+          echo "Detected Module '$module'"
+
+          # Check that it's not a provider
+          if [[ "$module" =~ [a-zA-Z0-9\-]+/terraform-module-[a-zA-Z0-9\-]+ ]]; then
+            gh issue comment $NUMBER -b "Failed validation: Likely a provider '$module'"
+            exit 1
+          fi
+
+          # Check that module matches expected pattern
+          if ! [[ "$module" =~ [a-zA-Z0-9\-]+/terraform-[a-zA-Z0-9]+-[a-zA-Z0-9\-]+ ]]; then
```

Comment:
This validation is probably good enough for now, but we might want to harden it further, to further align with the CLI.

For example, the target part should be limited to 64 character. You can see the regexes that the CLI uses for validation [here](https://github.com/opentofu/registry-address/blob/901b9ae4061a6e83e7db96e71b8d7088294930b3/module.go#L39-L40)

Reply:
Pulled in that dependency.